File: beecrowd/1042/1042 - Simple Sort .cs
using System;

class Program
{
	static void Main(string[] args)
	{
		string[] input;
		int a, b, c;

		input = Console.ReadLine().Split(' ');
		a = int.Parse(input[0]);
		b = int.Parse(input[1]);
		c = int.Parse(input[2]);

		int[] nums = { a, b, c };
		Array.Sort(nums);

		Console.WriteLine($"{nums[0]}\n{nums[1]}\n{nums[2]}");
		Console.WriteLine();
		Console.WriteLine($"{a}\n{b}\n{c}");
	}
}

File: beecrowd/1042/1042 - Simple Sort .cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main()
{
	int a, b, c;
	cin >> a >> b >> c;

	int nums[3] = {a, b, c};
	sort(nums, nums + 3);

	cout << nums[0] << "\n" << nums[1] << "\n" << nums[2] << endl;
	cout << endl;
	cout << a << "\n" << b << "\n" << c << endl;
	
	return 0;
}
